### PR TITLE
BIGTOP-3145: Crunch smoke tests implementation

### PR DESCRIPTION
**Pull request by guyuqi**
Add Crunch smoke tests.

### CODE REVIEW

File: bigtop-tests/smoke-tests/crunch/TestCrunchSmoke.groovy [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+import org.apache.bigtop.itest.shell.Shell
+import org.apache.bigtop.itest.TestUtils
+
+public class TestCrunchSmoke {
+  static String runnerScript = "hadoop jar"
+
+  static String crunchJar = System.getProperty(
+    "org.apache.bigtop.itest.crunch.smoke.crunch.jar",
+    "/usr/share/doc/crunch/crunch-examples-*job.jar");
```
Comment (evans-ye) [inline bigtop-tests/smoke-tests/crunch/TestCrunchSmoke.groovy]: Is this the right path to get example jar?

Comment (guyuqi) [reply bigtop-tests/smoke-tests/crunch/TestCrunchSmoke.groovy]: Yes, refer to: [install_crunch.sh](https://github.com/apache/bigtop/blob/master/bigtop-packages/src/common/crunch/install_crunch.sh#L96-L98)

Comment (guyuqi) [reply bigtop-tests/smoke-tests/crunch/TestCrunchSmoke.groovy]: I tested it on Ubuntu 16.04, and then launched this PR.
When I tested it based on CentOS-7 and fedora-26 today,  the puppet can not deploy rpm correctly.

`diff --git a/provisioner/docker/config_centos-7.yaml b/provisioner/docker/config_centos-7.yaml
index 49f86ae..aabd6b9 100644
--- a/provisioner/docker/config_centos-7.yaml
+++ b/provisioner/docker/config_centos-7.yaml
@@ -19,6 +19,6 @@ docker:

 repo: "http://repos.bigtop.apache.org/releases/1.3.0/centos/7/$basearch"
 distro: centos
-components: [hdfs, yarn, mapreduce]
-enable_local_repo: false
-smoke_test_components: [hdfs, yarn, mapreduce]
+components: [hdfs, yarn, mapreduce, crunch]
+enable_local_repo: true
+smoke_test_components: [crunch]
` 
The rpm is not from local though 'enable_local_repo' is true.

Comment (evans-ye) [reply bigtop-tests/smoke-tests/crunch/TestCrunchSmoke.groovy]: I reach the same conclusion. We need to fix the packaging first...

**Pull request closed without merging**